pkg/vadmin/start_db_vc.go

```diff
@@ -89,6 +89,12 @@ func (v *VClusterOps) genStartDBOptions(s *startdb.Parms, certs *HTTPSCerts) (vo
 	*opts.HonorUserInput = true
 
 	// timeout option
-	opts.StatePollingTimeout = v.VDB.GetRestartTimeout()
+	opts.StatePollingTimeout = new(int)
```

Comment:
Shouldn't we initialize this in vclusterops?

Reply:
Yes, we should. I will create a small JIRA to track this. 
You may help me merge this and let QA test. I’m curious to know if there’s any other problems (which may happen in other steps other than re_ip or start_db) when running kill pod during remove_node.